docs/decisions/0006-api-specification-for-enterprise-mfes.rst

```diff
@@ -0,0 +1,308 @@
+0006 API Specification for Enterprise Micro-frontends (MFEs)
+************************************************************
+
+Status
+======
+
+Accepted (March 2023)
+
+Context
+=======
+
+This document intends to outline which API endpoints exist or will exist in support of the
+Enterprise MFEs (e.g., frontend-app-learner-portal-enterprise) as well as other consumers,
+or service-to-service communication.
+
+API Endpoints
+=============
+
+GET Retrieve single, redeemable access policy for a course
+------------------------------------------------------
+
+**/api/v1/enterprise-customer/<enterprise_customer_uuid>/course/<course_key>/can_redeem/**
```

Comment:
Based on the proposed alternative below, this will change to 
```
/api/v1/policy/enterprise-customer/<enterprise_customer_uuid>/can_redeem?content_key=...&content_key=...
```
* Note, this supports multiple `content_key `params.
* `enterprise_customer_uuid` might actually make more sense as a param, too. TBD at implementation time.